### PR TITLE
Want to merge rspec branch to development branch

### PR DESCRIPTION
**Pull request by Kingobaino1**
**we write tests for the following methods**
- my_each
- my_each_with_index
- my_select
- my_all?
- my_any?
- my_none?
- my_map 
- my_inject
- my_count

### CODE REVIEW

File: spec/enumerable_spec.rb [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+    it 'false when no value is true on the hash' do
+      expect(hash.my_any? { |_k, v| v > 6 }).to eql FALSE
+    end
+  end
+
+  describe '#my_none?' do
+    it 'false when no value is true on the array' do
+      expect(array.my_none? { |n| n > 2 }).to eql FALSE
+    end
```
Comment (ghost) [inline spec/enumerable_spec.rb]: Improve the statement for this test. `my_none?` method returns `true` when no value in the array is `true`. Your statement says it returns `false` instead.

File: spec/enumerable_spec.rb [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+    it 'true when any value is true on the array' do
+      expect(array.my_none? { |n| n > 6 }).to eql TRUE
+    end
```
Comment (ghost) [inline spec/enumerable_spec.rb]: The statement should say, it returns` false` when any value is `true` on the array. Yours says it returns `true`.

File: spec/enumerable_spec.rb [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+    it 'false when no value is true on the range' do
+      expect(range.my_none? { |n| n > 2 }).to eql FALSE
+    end
```
Comment (ghost) [inline spec/enumerable_spec.rb]: Should say, it returns `true` instead of `false`.

File: spec/enumerable_spec.rb [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+
+    it 'false when any value is true on the range' do
+      expect(range.my_none? { |n| n > 6 }).to eql TRUE
+    end
+
+    it 'false when no value is true on the hash' do
+      expect(hash.my_none? { |_k, v| v > 2 }).to eql FALSE
+    end
```
Comment (ghost) [inline spec/enumerable_spec.rb]: This statement should say, it returns `true` when no value is true on the hash. Yours says `false`.

File: spec/enumerable_spec.rb [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+    it 'true when any value is true on the hash' do
+      expect(hash.my_none? { |_k, v| v > 6 }).to eql TRUE
+    end
+  end
```
Comment (ghost) [inline spec/enumerable_spec.rb]: The statement should say,  '`false` when any value is true on the hash'.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

**Hello team,**

**Great work on your project. Your code looks clean. There are just a few fixes required before the project can be approved.**

1). **There is a detected offense by rubocop.**

rubocop

2). **Check a few comments below.**

### Happy coding 😃.

**PS: You can reach me on slack (Rose Wanjohi).**

**Comment by sergioobolevich** [comment]
> # Status: Changes Required
> **Hello team,**
> 
> **Great work on your project. Your code looks clean. There are just a few fixes required before the project can be approved.**
> 
> 1). **There is a detected offense by rubocop.**
> 
> rubocop
> 
> 2). **Check a few comments below.**
> 
> ### Happy coding 😃.
> **PS: You can reach me on slack (Rose Wanjohi).**

Dear @blackpintz , thanks for your review.
We have realized that linters from Github is not the same configuration as the linters you are using on your computer.
Therefore, is difficult to fix the error as we are not getting it from Github.
So please, we ask you if possible to clarify how to fix it, or we ask you to pass on this error as we were unable to find how to fix it.

Thanks in advance.

**Review by kubilaycaglayan — APPROVED** [review]
### Hello Team 👋
# Status: Project Approved :green_circle: 

Great job! Thanks for making the required changes! 👏👍
All your tests are passing :heavy_check_mark: :heavy_check_mark: 

- [ ]  [OPTIONAL] You can add tests for `multiply_els` method.

With all my respect 🙏
> Kubilay Caglayan | Technical Support Engineer

**Pull request merged**